scripts/modal.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function installModal () {
+  // Get the modal
+  const modal = document.getElementById('myModal')
+
+  // Get the <span> element that closes the modal
+  const span = document.getElementsByClassName('close')[0]
```

Comment:
I think it would be good to scope this selector:

```suggestion
  const span = modal.getElementsByClassName('close')[0]
```